### PR TITLE
Add '-r' rm file ability

### PR DESCRIPTION
**Pull request by TerraTech**
1. strip-docker-image: Fix bug where multiple '-f' were not appending to $FILES
1. Add -r' option to remove files from final image
1. misc mcedit syntax highlighting fix where it gets confused with multiple quotes in an $() construct

As an aside, can you turn on issues for your fork?  It seems that your fork is the best quality I've seen thus far as the official repo seems to have become neglected.

Also, if you need for me to squash these commits, just let me know.

### CODE REVIEW

**Comment by TerraTech** [comment]
Note: I chose '-r' over '-d' as I've seen in other forks '-d' was being used to specify reading from a Dockerfile.

**Comment by lkwg82** [comment]
> As an aside, can you turn on issues for your fork?

of course

File: bin/strip-docker-image-export [hunk 1]
```diff
@@ -77,7 +78,10 @@ function parse_commandline() {
 		    PACKAGES="$PACKAGES $OPTARG"
 		    ;;
 		f)
-		    FILES="$FILES $(printf "${OPTARG}")"
+		    FILES+=" $(printf "${OPTARG}")"
```
Comment (lkwg82) [inline bin/strip-docker-image-export:83]: is this by intention?

**Comment by lkwg82** [comment]
plz update the readme.md as well

File: bin/strip-docker-image-export (hunk 1 above)
Comment (TerraTech) [reply bin/strip-docker-image-export:83]: Well it was, until you fixed argument handling in: https://github.com/lkwg82/strip-docker-image/commit/fca7cf7faca2488a6496039edaf179ee38109c83

My pull request pre-dated that commit, and I had utilized bash's += operator...

**Comment by TerraTech** [comment]
Given that all this started with fixing '-f', it snowballed into my adding '-r' as well (for a particular project I was currently working on).  Are you interested in adding 'remove' ability?  I also had updated the readme.me file and added '-r' to it and I'm not sure what else is needed to be added.

File: bin/strip-docker-image-export (hunk 1 above)
Comment (lkwg82) [reply bin/strip-docker-image-export:83]: aah, sry oversaw it.

**Review by lkwg82 — CHANGES_REQUESTED** [review]
just a test

**Review by lkwg82 — APPROVED** [review]

**Review by lkwg82 — CHANGES_REQUESTED** [review]
```bash
# strip-docker-image -i ubuntu -t ubuntu-stripped-tar -p tar
/mybin/strip-docker-image-export: line 81: printf: -r: invalid option
printf: usage: printf [-v var] format [arguments]
tar: Removing leading `/' from member names
Sending build context to Docker daemon 3.281 MB
Step 1 : FROM scratch
 ---> 
Step 2 : ADD export /
 ---> fa1999b98b1c
Removing intermediate container 0639cdf49265
Step 3 : CMD /bin/bash
 ---> Running in 2897936655af
 ---> 9c1faca3e887
Removing intermediate container 2897936655af
Successfully built 9c1faca3e887
```
Something went wrong

**Comment by TerraTech** [comment]
@lkwg82 it is a simple fix to the join() function, however I'm working on rebasing to your latest commits.

**Comment by TerraTech** [comment]
@lkwg82 I've rebased and force pushed which fixes the errant join assumption and only contains the new _rm_ ability.  I removed the '-f' fix as your commit resolves that issue: https://github.com/lkwg82/strip-docker-image/commit/fca7cf7faca2488a6496039edaf179ee38109c83

**Review by lkwg82 — APPROVED** [review]

**Pull request merged**